tools/accuracy_checker/accuracy_checker/evaluators/module_evaluator.py

```diff
@@ -36,6 +36,7 @@ def from_configs(cls, config, *args, **kwargs):
         return cls(load_module(module, python_path).from_configs(module_config, *args, **kwargs), config)
 
     def process_dataset(self, stored_predictions, progress_reporter, *args, **kwargs):
+        kwargs['processing_info'] = self._internal_module.get_processing_info(self._config)
```

Comment:
no need to provide it into evaluator, you can get it inside internal module directly